src/scenario/scenario.c

```diff
@@ -7,36 +7,124 @@
 #include "game/save_version.h"
 #include "game/settings.h"
 #include "scenario/data.h"
+#include "scenario/request.h"
 
 struct scenario_t scenario;
 
+static struct {
+    int size;
+    int start_info;
+    int old_filler_1;
+    int invasions_part1;
+    int funds_enemy;
+    int map_size;
+    int briefing;
+    int old_filler_2;
+    int image;
+    int herds;
+    int demand_changes_part1;
+    int price_changes_part1;
+    int glademp;
```

Comment:
This `glademp` took me a long while to figure out. Please make the variable more descriptive. I think you should split it into `gladiator_revolt` and `emperor_change` actually.

Reply:
Done.